public/components/marketing-forms/5.html

```diff
@@ -0,0 +1,67 @@
+<!--
+  Heads up! 👋
+
+  Plugins:
+    - @tailwindcss/forms
+-->
+
+<section class="w-full space-y-4 bg-white md:space-y-6">
+  <div>
+    <h2 class="text-2xl font-bold text-gray-900">Profile Settings</h2>
+    <p class="text-sm text-gray-700">Manage your account settings</p>
+  </div>
+  <form action="" class="border-t-2 pt-4 md:pt-6">
+    <div class="space-y-4 divide-y md:space-y-6">
+      <div class="grid gap-4 md:grid-cols-6">
```

Comment:
Let's add `grid-cols-1` as well, I've ran into issues when expecting `grid-cols-1` to auto apply.